src/components/screens/admin-vacation-management/UserRow.tsx

```diff
@@ -0,0 +1,39 @@
+import type React from 'react';
+import { TableRow, TableCell, IconButton } from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import type { User } from '../../../types/index';
+
+interface UserRowProps {
+  user: User;
+  onEdit: (user: User) => void;
+}
+
+const UserRow: React.FC<UserRowProps> = ({ user, onEdit }) => {
+  const currentYear = new Date().getFullYear().toString();
+
+  const getDays = (arr: string[] | undefined, year: string) => {
```

Comment:
Add Docs to clarify what this function is doing.
All functions should have some documentation/comment